scripts/zones/Cloister_of_Flames/bcnms/sugar-coated_directive.lua

```diff
@@ -21,7 +21,8 @@ battlefield_object.onBattlefieldLeave = function(player, battlefield, leavecode)
     if leavecode == xi.battlefield.leaveCode.WON then
         local _, clearTime, partySize = battlefield:getRecord()
         local arg8 = (player:hasCompletedMission(xi.mission.log_id.ASA, xi.mission.id.asa.SUGAR_COATED_DIRECTIVE)) and 1 or 0
-        player:startEvent(32001, battlefield:getArea(), clearTime, partySize, battlefield:getTimeInside(), 1, battlefield:getLocalVar("[cs]bit"), arg8)
+        player:startEvent(32001, battlefield:getArea(), clearTime, partySize, battlefield:getTimeInside(), 201, battlefield:getLocalVar("[cs]bit"), arg8)
+        player:setCharVar("Mission[11][3]Ifrit", 2)
```

Comment:
Please see the pattern we've been using for mission BCNMs.  (search the `battlefieldWin` localvar for examples)